src/views/DandisetLandingView/DandisetUnembargo.vue

```diff
@@ -0,0 +1,150 @@
+            it will be public forever.
+            <br><br>
+            Note: this may take a while if your dandiset is large.
+          </span>
+        </v-card-text>
+        <v-card-text>
+          Would you like to proceed?
+        </v-card-text>
+
+        <v-card-actions>
+          <v-btn
+            color="error"
+            depressed
+            @click="release()"
+          >
+            Yes
+          </v-btn>
+          <v-btn
+            depressed
+            @click="showWarningDialog = false"
+          >
+            No, take me back
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+    <v-row
+      class="mb-4"
+      no-gutters
+    >
+      <v-tooltip
+        left
+        :disabled="!unembargoing"
+      >
+        <template #activator="{ on }">
+          <div
+            class="px-1 pt-3"
+            style="width: 100%"
+            v-on="on"
+          >
+            <v-btn
+              block
+              color="info"
+              depressed
+              :disabled="unembargoing"
+              @click="release()"
+            >
+              {{ unembargoing ? 'Releasing' : 'Release' }}
+              <v-spacer />
+              <v-icon>mdi-lock-open</v-icon>
+            </v-btn>
+          </div>
+        </template>
+        <span v-if="unembargoing">This dandiset is being released, please wait.</span>
+      </v-tooltip>
+    </v-row>
+    <v-row>
+      <v-subheader class="mb-2 black--text text-h5">
+        This Version
+      </v-subheader>
+    </v-row>
+    <v-row
+      class="pa-2 mb-5 text-body-2 align-center"
+      style="border-top: thin solid rgba(0, 0, 0, 0.12);
+             border-bottom: thin solid rgba(0, 0, 0, 0.12);
+             text-align: center;"
+    >
+      <v-col
+        :cols="$vuetify.breakpoint.md ? 12 : 4"
+        style=""
+      >
+        {{ formatDate(currentDandiset.modified) }}
+      </v-col>
+      <v-col>
+        <h3>{{ currentDandiset.version.toUpperCase() }}</h3>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, ref } from '@vue/composition-api';
+import moment from 'moment';
+import { dandiRest } from '@/rest';
+import store from '@/store';
+
+function formatDate(date: string): string {
+  return moment(date).format('ll');
+}
+
+export default defineComponent({
+  name: 'DandisetUnembargo',
+  setup() {
+    const currentDandiset = computed(() => store.state.dandiset.dandiset);
+    const unembargoing = computed(() => currentDandiset.value?.dandiset.embargo_status === 'UNEMBARGOING');
+    const showWarningDialog = ref(false);
+
+    async function release() {
+      if (currentDandiset.value) {
+        // Display the warning dialog before releasing
+        if (!showWarningDialog.value) {
+          showWarningDialog.value = true;
+          return;
+        }
+
+        await dandiRest.publish(currentDandiset.value.dandiset.identifier);
```

Comment:
```suggestion
        await dandiRest.unembargo(currentDandiset.value.dandiset.identifier);
```
The `publish` endpoint is not being recycled, there should be a new method just for unembargoing.

Reply:
https://github.com/dandi/dandiarchive/pull/1021/commits/a24250359265894b2a90a0946f3b130c8b6069f9